### PR TITLE
Changed default directory for generating steps file to the same directory as the .feature file

### PR DESCRIPTION
**Pull request by 304NotModified**
First it will try to find the folder <projectfolder>/StepDefinitions (current code)
If not found, then use the directory where the .feature file lives

fixes https://github.com/SpecFlowOSS/SpecFlow/issues/1638

Tested it locally with VS2019 multiple times. 

Todo

- [ ] update changelog

### CODE REVIEW

File: TechTalk.SpecFlow.VsIntegration.Implementation/UI/GenerateStepDefinitionSkeletonForm.cs [hunk 1]
```diff
@@ -85,7 +85,7 @@ public GenerateStepDefinitionSkeletonForm(string featureTitle, StepInstance[] st
             classNameTextBox.Text = string.Format("{0} Steps", featureTitle).ToIdentifier();
 
             styleComboBox.BeginUpdate();
-            var styles = Enum.GetValues(typeof (StepDefinitionSkeletonStyle)).Cast<StepDefinitionSkeletonStyle>()
+            var styles = Enum.GetValues(typeof(StepDefinitionSkeletonStyle)).Cast<StepDefinitionSkeletonStyle>()
```
Comment (304NotModified) [inline TechTalk.SpecFlow.VsIntegration.Implementation/UI/GenerateStepDefinitionSkeletonForm.cs:88]: FYI: change by format document

File: TechTalk.SpecFlow.VsIntegration.Implementation/UI/GenerateStepDefinitionSkeletonForm.cs [hunk 2]
```diff
@@ -97,7 +97,10 @@ public GenerateStepDefinitionSkeletonForm(string featureTitle, StepInstance[] st
 
             defaultFolder = Path.Combine(VsxHelper.GetProjectFolder(specFlowProject), "StepDefinitions");
             if (!Directory.Exists(defaultFolder))
-                defaultFolder = VsxHelper.GetProjectFolder(specFlowProject);
+            {
+                var currentFeatureFilePath = dte.ActiveDocument.FullName;
```
Comment (304NotModified) [inline TechTalk.SpecFlow.VsIntegration.Implementation/UI/GenerateStepDefinitionSkeletonForm.cs:101]: If I need some try-catch here, let me know (not need from testing)

**Comment by SabotageAndi** [comment]
Thanks for your contribution to SpecFlow. If you would like us to send you some SpecFlow stickers as a thank you, please fill out the form [here](https://specflow.org/contributer-package/).

**Comment by 304NotModified** [comment]
Thanks! I like that :)

**Pull request merged**